packages/rrdom/src/util.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Get the content document of an iframe.
+ * Catching errors is necessary because some older browsers block access to the content document of a sandboxed iframe.
+ */
+export function getIFrameContentDocument(iframe?: HTMLIFrameElement) {
+  if (iframe) {
```

Comment:
l: Can we skip this? Since we already try-catch this, we can safe a tiny amount of bytes by just doing:

```ts
try {
  return iframe!.contentDocument
} catch {
  // noop
}
```

?